src/main/test/at/ac/fhcampuswien/fhmdb/models/MovieTest.java

```diff
@@ -70,4 +72,15 @@ public void throws_an_exception_if_Movie_B_is_null() {
         }
     }
 
+    @Nested
+    public class InitializeMovies {
+        @Test
+        public void returns_a_list_of_movies_with_more_than_0_movies() {
+            List<Movie> movies;
+
+            movies = Movie.initializeMovies();
+
+            Assertions.assertTrue(movies.size() > 0);
```

Comment:
Can you please add some assertions that the list contains at least a `Movie` object, which has a title, a description and a `Genre`?

Reply:
If these are required fields, I would put them to the constructor parameters. 

Title and Description are already there, so they cannot be null. Maybe empty if the string is empty.

Maybe we should add Genre to the constructor?